dotcom-rendering/src/amp/lib/region-classes.ts

```diff
@@ -1,5 +1,10 @@
 import { css } from '@emotion/react';
 
+// Array of possible ad regions
+export const adRegions = ['UK', 'US', 'AU', 'INT'] as const;
```

Comment:
We add an additional 'UK' region (previously it was just 'US, 'AU' and 'ROW' for regional ads) as the new Prebid servers have parameters that vary between UK and international.